site/content/docs/commands/job-logs.en.md

```diff
@@ -0,0 +1,62 @@
+# job logs
+```console
+$ copilot job logs
+```
+
+## What does it do?
+
+`copilot job logs` displays the logs of a deployed service.
+
+## What are the flags?
+
+```  
+  -a, --app string              Name of the application.
+      --end-time string         Optional. Only return logs before a specific date (RFC3339).
+                                Defaults to all logs. Only one of end-time / follow may be used.
+  -e, --env string              Name of the environment.
+      --follow                  Optional. Specifies if the logs should be streamed.
+  -h, --help                    help for logs
+      --include-state-machine   Optional. Include logs from the state machine executions.
+      --json                    Optional. Output in JSON format.
+      --last int                Optional. The number of executions of the scheduled job for which
+                                logs should be shown. (default 3)
+      --limit int               Optional. The maximum number of log events returned. Default is 10
+                                unless any time filtering flags are set.
```

Comment:
Out of curiosity, is the default actually 10? It seems to be set to 0 in the code but I haven't read the code close enough to figure out whether it gets defaulted to 10 later.

Reply:
This is the one https://github.com/aws/copilot-cli/blob/e1083b91cd96a0c3123789ac8638b6e181540ffe/internal/pkg/logging/workload.go#L23